podcasts/End of Year/Stories/IntroStory.swift

```diff
@@ -99,14 +99,16 @@ struct IntroStory: ShareableStory {
 
 private struct TwentyThreeParallaxModifier: ViewModifier {
     @Environment(\.accessibilityReduceMotion) private var reduceMotion
-    @StateObject var manager: MotionManager = .init()
+    @StateObject var manager: MotionManager = .init(relativeToWhenStarting: true)
     var rollMultiplier: Double = 4
     var pitchMultiplier: Double = 40
 
+    private let rollAndPitchBoundary = -1.4..<1.5
```

Comment:
Even with these boundaries, I feel that if you do crazy tilting the numbers jump crazy. However, I'm considering this an edge case for now 🤷 

Any ideas are very appreciated though.

Reply:
This is oddly annoying and difficult to fix, and something I spent far too long trying to. 

A problem with this solution is if the user is lying in bed, with their phone upside then the effect breaks. Unfortunately I don't have a better solution though. 

https://github.com/Automattic/pocket-casts-ios/assets/793774/57d5fd7c-6f76-46d3-8b22-65dd5c426d2c

Reply:
I might be wrong here but I think this issue is caused even without the changes on this PR?

So at least we can have the "jump" issue fixed with it.

Reply:
true!!